docs/corpora_schema.md

```diff
@@ -34,100 +35,110 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
     combined before submission. For example, raw counts will be summed and logged counts will be exponentiated, summed, and logged.
-*   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a
-    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
+    This is needed for the explorer to function. When a user requests gene expression information, the explorer needs to be able to unambiguously return a
+    single value.
+*   **No PII**. Curators agree to this requirement as part of the data submission policy. However, it is not strictly enforced in our validation tooling because it
+    is difficult for software to predict what is and is not PII. It is up to the submitter to ensure that no metadata can be personally identifiable: no names,
+    dates of birth, specific locations, etc. There's a [list](https://docs.google.com/document/d/1sboOmbafvMh3VYjK1-3MAUt0I13UUJfkQseq8ANLPl8/edit).
 
 ### Matrix Layers
 
-The count matrix itself can exist in several forms. Generally, there is a "raw" count matrix without scaling, filtering, normalization, etc. And there is
-a "final" matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices. We require that these matrices
-have the same dimensions, so the raw count matrix should include the same cells and genes as the final.
-
-These different transformations of the count matrix are called "layers" in several file formats. Submissions to the Data Portal must contain the "raw"
-layer. Note that AnnData objects have an attribute called [raw](https://anndata.readthedocs.io/en/latest/anndata.AnnData.raw.html#anndata.AnnData.raw). This
-could be a natural place to store the raw count matrix, but as long as the locations of the layers are specified, it can go anywhere.
-
-
-### Schema Version
-
-Datasets in the Data Portal must store the version of the schema they follow (that is, the version of this document) as
-well as the version of the particular encoding used. The encoding is documented
-[elsewhere](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) and describes techincal details
-of how the schema should be serialized in a particular file format.
-
-**Field name**|**Constraints**
-:--|:--
-corpora_schema_version|Follows [SemVer](https://semver.org/) conventions
-corpora_encoding_version|Follows [SemVer](https://semver.org/) conventions
+cellxgene's data requirements are tailored to optimize data reuse. Because each assay has different characteristics, our requirements differ by assay type. In general,
+cellxgene requires submission of "raw" data suitable for computational reuse, when a standard form exists, and strongly suggests that a "final" matrix suitable for
```

Comment:
If a contributor does not provide a "final" matrix, what does cxg use for the visualization? Does cxg create the "final" matrix from the "raw" matrix normalized using a standard process?

Reply:
> Does cxg create the "final" matrix from the "raw" matrix normalized using a standard process?

No - we would use the raw data that is submitted. This is why "final" is strongly recommended.